backend/src/main/java/co/inventorsoft/scripty/service/MockRequestService.java

```diff
@@ -0,0 +1,99 @@
+package co.inventorsoft.scripty.service;
+
+import co.inventorsoft.scripty.exception.ApplicationException;
+import co.inventorsoft.scripty.model.dto.MockRequestDto;
+import co.inventorsoft.scripty.model.entity.MockRequestEntity;
+import co.inventorsoft.scripty.repository.MockRequestRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import java.nio.charset.Charset;
+import java.util.Arrays;
+import java.util.UUID;
+
+/**
+ * @author A1lexen
+ */
+
+@Service
+public class MockRequestService {
+    private enum Methods {
+        POST("post"),
+        PUT("put"),
+        GET("get");
+
+        private String method;
+
+        Methods(String method) {
+            this.method = method;
+        }
+
+        public static boolean contains(String str) {
+            return Arrays.stream(values()).anyMatch(m -> m.method.equals(str.toLowerCase()));
+        }
+    }
+
+    private enum ContentTypes {
+        APPLICATION_JSON("application/json"),
+        APPLICATION_XML("application/xml"),
+        TEXT_PLAIN("text/plain"),
+        TEXT_HTML("text/html");
+
+        private String type;
+
+        ContentTypes(String type){
+            this.type = type;
+        }
+        public static boolean contains(String str) {
+            return Arrays.stream(values()).anyMatch(m -> m.type.equals(str.toLowerCase()));
+        }
+    }
+
+    private MockRequestRepository requestRepository;
+
+    @Autowired
+    MockRequestService(MockRequestRepository requestRepository ) {
+        this.requestRepository = requestRepository;
+    }
+
+    public String createNewRequest(MockRequestDto requestDto) {
+        MockRequestEntity request = new MockRequestEntity();
+
+        MediaType.parseMediaType(requestDto.getContentType());
+        if ((requestDto.getStatus() < 100 ||  requestDto.getStatus() > 599)) throw new ApplicationException("Invalid Status Code input!", HttpStatus.BAD_REQUEST);
```

Comment:
This is a part of validation process. Please, create custom constraints or reuse existing one from Hibernate validator to express this inside `MockRequestDto`